hicommonwealth/commonwealth
# Safe URI Decoding + Profile Bio Decoding

## Link to Issue
Closes: #9264

## Description of Changes
- Uses `getDecodedString` function instead of `decodeURIComponent` to support both decoded and encoded strings
- Adds a function to the decode script to decode profile bios
- Fixes the `updateUser` command to fix bio decoding

## Test Plan
- Generate a comment notification
  - Create a thread from user A
  - Comment on the thread from user B
  - Check user A notification format
- View a profile with a bio i.e. http://localhost:8080/profile/id/3193
- Run `pnpm ts-exec scripts/decode-db-content.ts profiles` from `packages/commonwealth`
- View the profile bio again ensuring it retained its markdown formatting
- Edit your own profile bio multiple times (need to ensure format after edit is the same)



## Deployment Plan

1. 

## Other Considerations

- Required PR for release 1.6.1
